config/environments/test.rb

```diff
@@ -22,7 +22,7 @@
   config.consider_all_requests_local       = true
   config.action_controller.perform_caching = true
 
-  config.cache_store = :redis_cache_store, { url: ENV.fetch("REDIS_URL", "redis://localhost:6379/0") }
```

Comment:
The previous set up was using the same redis db for testing and dev, this is an issue if you are running redis for a local server and run any tests that utilize rails cache. Redis comes with 15 separate db's so a simple fix is to set the testing environment cache to use a different db index